{"name": "Логин с неверным логином и паролем", "status": "passed", "start": 1730030500110, "stop": 1730030500732, "uuid": "de499ba3-6850-42bd-8bc6-da19d4be82ee", "historyId": "fbeae77312463979bb58e896f211518f", "testCaseId": "fbeae77312463979bb58e896f211518f", "fullName": "tests.test_user_login.TestUserLogin#test_login_with_invalid_credentials", "labels": [{"name": "feature", "value": "Логин пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "Liana_M"}, {"name": "thread", "value": "18968-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}